samples/snippets/src/main/java/IdTokenFromImpersonatedCredentials_IAM.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
More context on why the sample will be deleted:

This sample uses the older IAM Apiary library. And this might not be the best way to obtain a IdToken. Rather, use the OAuth library which provides the appropriate methods for the same.